### DISCUSSION TITLE
Idée pour bot python?

### DISCUSSION THREAD

**Discussion by Thomass789** [drknzz/GitHub-Achievements#63] [Q&A]
Idée pour bot python?

**Comment by caaiopereira** [comment]
I am answering, Thanks!

**Comment by S1i** [comment]
If you mean discord bot then make a minigames bot with snakes & ladders, chess, rock paper scissors etc

**Comment by OwaisAzad** [comment]
With the help of a python bot, you could suggest tasks such as automating repetitive tasks, generating reports, monitoring social media, or sending automated notifications.

**Comment by CharlesCodebabel** [comment]
Il existe un réseau social qui s'appelle Koo, je crois qu'il n'y a pas encore de bot Koo...
https://www.kooapp.com

**Comment by DanielR-spec** [comment]
Thank u for ur support :100:

**Comment by bettyc0der** [comment]
Big 🧠

**Comment by Boutzi** [comment]
Tu peux faire un bot discord, ou meme un jeu discord (c'est en vogue en ce moment)
EN : You can do a discord bot, or even a discord game!
